plugins/module_utils/foreman_helper.py

```diff
@@ -220,7 +220,7 @@ def __init__(self, argument_spec, **kwargs):
         if 'verify_ssl' in self._params:
             self.warn("Please use 'validate_certs' instead of deprecated 'verify_ssl'.")
 
-        self.task_timeout = 60
+        self.task_timeout = 240
```

Comment:
You can adjust that timeout in the module itself.

See https://github.com/theforeman/foreman-ansible-modules/blob/4ab1fbea7140302b818cd2674e8d95cd7612f7b2/plugins/modules/katello_sync.py#L113